src/Mesh/Interpolation.jl

```diff
@@ -511,9 +511,9 @@ struct InterpolationCubedSphere{FT <: AbstractFloat,
         nblck = nhor * nhor * nvert
         Δh = 2 / nhor                               # horizontal grid spacing in unwarped grid
 
-        lat_grd  = range(lat_min,  lat_max,  step=lat_res)
-        long_grd = range(long_min, long_max, step=long_res)
-        rad_grd  = range(rad_min,  rad_max,  step=rad_res)
+        lat_grd  = collect(range(lat_min,  lat_max,  step=lat_res))
```

Comment:
lets use degrees (and the correct ranges) everywhere, i.e. set
```julia
lat_min,   lat_max = FT(-90), FT(90)
long_min, long_max = FT(-180), FT(180)
```
and use the "degree" trig functions (`sind`, `cosd`, etc)